modules/nodes-descheduler-installing.adoc

```diff
@@ -33,7 +33,15 @@ endif::[]
 . Create a descheduler instance.
 .. From the *Operators* -> *Installed Operators* page, click the *Kube Descheduler Operator*.
 .. Select the *Kube Descheduler* tab and click *Create KubeDescheduler*.
-.. Edit the settings as necessary. You can configure the descheduler profiles and customizations now or after it has been created.
+.. Edit the settings as necessary.
+... Expand the *Profiles* section to select one or more profiles to enable. The `AffinityAndTaints` profile is enabled by default. Click *Add Profile* to select additional profiles.
++
+[NOTE]
+====
+Do not enable both `TopologyAndDuplicates` and `SoftTopologyAndDuplicates` because they will conflict with each other.
```

Comment:
```suggestion
Do not enable both `TopologyAndDuplicates` and `SoftTopologyAndDuplicates`. Enabling both results in a conflict.
```

Just a suggestion, I was struggling with the use of "they". Not required to change it though.

Reply:
Great suggestion! I use this phrasing in another module, and will change it there as well :)